File: Q1.js
// Rainfall Problem

const arr = [];
let sum = 0;
let c = 0.0;
let res = 1;
function rainfall(arr) {
  if (arr.length === 0) return console.log("No rain");
  for (let i of arr) {
    if (i > 0) {
      c++;
      sum += i;
    }
  }
  //   c = Math.max(c++)
  res = sum / c;
  return console.log(res);
}

rainfall(arr);

File: q1.py
def rainfall(r):
    total = 0
    count = 0.0
    for record in r:
        if record>0:
            total += record
            count += 1.0
    if total>0:
        return (total/count)
    else:
        return("No Rain")
    
    
# a = [1,2,3,4]
# b = [-1,1,2,3]

# print(rainfall(a))
# print(rainfall(b))